modules/host/storageobligations.go

```diff
@@ -287,7 +287,7 @@ func (h *Host) queueActionItem(height types.BlockHeight, id types.FileContractID
 	// Sanity check - action item should be at a higher height than the current
 	// block height.
 	if height <= h.blockHeight {
-		h.log.Critical("action item queued improperly")
+		h.log.Println("action item queued improperly")
```

Comment:
Changed to a println from a critical, because we've known about this error for a very long time and we have a plan to fix it eventually, but for now we're focused 100% on making the renter more scalable and getting filesharing live.